### ISSUE TITLE
Move repository to PHPCSStandards organisation

### ISSUE THREAD

**Issue by Potherca** [PHPCSStandards/composer-installer#146]
As mentioned before in #113 it is desirable to move this project to an organization that has more contributors and higher visibility.

Currently [PHPCSStandards](https://github.com/phpcsstandards/) looks like the best match.

This issue proposes to move the repository for this project from the Dealerdirect organisation to PHPCSStandards.

Opinions?

**Comment by Potherca** [comment]
As this motion has been seconded by both other active maintainers, lets make the move!

**Comment by jrfnl** [comment]
The one thing we should still discuss about this - should the package also be renamed to `phpcsstandards/...`.

The downside of that is that 1) the stats won't be retained and 2) it will be quite a while before all users have switched over.
The upside of it is that we won't be dependent on the DealerDirect organisation continuing to allow us to manage the package on Packagist.

**Comment by Potherca** [comment]
My current thinking is (more or less) that we _should_ rename to `phpcsstandards` with release v1.0.
It might be less worrisome that only the statistics for pre-1.0 will be lost that way :grin:  

Regarding renaming, we might want to take it one step further and go from `Dealerdirect/phpcodesniffer-composer-installer` to `phpcsstandards/composer-installer` (as the "phpcodesniffer" part is implied in "phpcsstandards").

**Comment by Potherca** [comment]
Also... Do we have any reason for not pressing the "transfer" button _right now_?

[![image](https://user-images.githubusercontent.com/195757/153216382-1d28c662-c6a8-416a-8a03-3df48ce3eab0.png)](https://github.com/Dealerdirect/phpcodesniffer-composer-installer/settings)

**Comment by jrfnl** [comment]
> My current thinking is (more or less) that we _should_ rename to `phpcsstandards` with release v1.0. It might be less worrisome that only the statistics for pre-1.0 will be lost that way 😁

Fair enough, clean slate and all.

For the record, current stats are pretty impressive already:

![image](https://user-images.githubusercontent.com/663378/153220908-39d3e26e-aa87-4ef7-9a2e-6c5e49b2caa9.png)

> Regarding renaming, we might want to take it one step further and go from `Dealerdirect/phpcodesniffer-composer-installer` to `phpcsstandards/composer-installer` (as the "phpcodesniffer" part is implied in "phpcsstandards").

I agree, sounds like a good plan. On that note, we can rename the repo to `composer-installer` as well after the move.


> Also... Do we have any reason for not pressing the "transfer" button right now?

Go for it! I'll handle the accepting on the other end.

**Comment by GaryJones** [comment]
> we can rename the repo to composer-installer as well after the move.

Does GitHub auto-redirect on a change like that?

**Comment by Potherca** [comment]
> > Also... Do we have any reason for not pressing the "transfer" button right now?
> 
> Go for it! I'll handle the accepting on the other end.

Looks like I'm not allowed:
```
You don’t have the permission to create public repositories on PHPCSStandards 
```

**Comment by jrfnl** [comment]
> > > Also... Do we have any reason for not pressing the "transfer" button right now?
> > 
> > 
> > Go for it! I'll handle the accepting on the other end.
> 
> @jrfnl Looks like I'm not allowed:
> 
> ```
> You don’t have the permission to create public repositories on PHPCSStandards 
> ```

@Potherca Huh ? That sounds weird - In my view, a transfer request should be allowed to be initiated by any admin of a repo with the acceptance being the responsibility of the organisation on the other end (with a mail notifying them there is a transfer request to accept).

May be something weird on GH's end.

I'll give it a try myself. Didn't want to do that before as for "auditability/transparency" it is, of course, better if the transfer is done by multiple people showing that there is consensus about it.

As a side-note: I have already invited both you as well as @mjrider to the PHPCSStandards organisation, but looks like neither of you have accepted yet.

**Comment by jrfnl** [comment]
> I'll give it a try myself.

Looks like that worked. Shall I rename the repo to `composer-installer straight away as well ?

**Comment by jrfnl** [comment]
> > Shall I rename the repo to `composer-installer straight away as well ?
> Yes please, the sooner the better IMHO.

Done!

**Comment by GaryJones** [comment]
I had this issue open before the change was made, and I can confirm that GitHub successfully redirected me back to this issue when reloading the page. 👍🏻

**Comment by jrfnl** [comment]
@GaryJones Thanks for confirming. The issue will automatically be closed when #158 gets merged, which updates the references to the repo location in all relevant places in the code base.

**Comment by jrfnl** [comment]
@mjrider Looks like I don't get a notification mail when an invite to an organisation is accepted. Could you please leave a note here when you have ? I will then add you to the "composer plugin admins" team, which should give you admin access to the repo again.
